### PR TITLE
Update arguments sent in slash command request

### PR DESCRIPTION
**Pull request by jasonblais**
Please confirm this is correct, based off of https://community.mattermost.com/core/pl/ae4yommjpprippfdm3bwj8moiw

### CODE REVIEW

**Comment by sadohert** [comment]
I had a peek at this as I was doing some custom slash command work.  Should those ampersands "&" appear at the end of the first few POST/form parameters?  Doesn't seem like it on my setup.

File: site/content/integrate/slash-commands/_index.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ text=asd&
 token=okwexkjpe7ygb8eq1ww58t483w&
 user_id=aoa1agao6t8fmx3ikt1j9w5ybw&
 user_name=somename
+root_id=be4yommjpprippfdm3bwj8moiw
+parent_id=syrsy8o84jbgdny5efb9hc4ezy
+trigger_id=3jt7dd34x3r9prj7e6eq7f7f4o
+command=test
```
Comment (crspeller) [inline site/content/integrate/slash-commands/_index.md]: Here is one I generated from a slash command with target http://127.0.0.1:8080 and trigger world `example` on my local instance.
```http
POST / HTTP/1.1
Host: 127.0.0.1:8080
User-Agent: mattermost-5.9.0
Content-Length: 548
Accept: application/json
Authorization: Token nezum4kpu3faiec7r7c5zt6tfy
Content-Type: application/x-www-form-urlencoded
Accept-Encoding: gzip

channel_id=i3bb9xfyqt8rtbyshmyhgsj16c&
channel_name=town-square&
command=%2Fexample&
response_url=http%3A%2F%2F10.0.0.5%3A8065%2Fhooks%2Fcommands%2Fzozc1xwxybdedeyz8djwjpngny&
team_domain=rrrr&
team_id=tsb8crrn5tgqtedpkt81b4tcya&
text=&
token=nezum4kpu3faiec7r7c5zt6tfy&
trigger_id=NG1kM3lyN2NqYmQxcGNyc2s0Nmo5em0xb2M6azF4NGFxZGp5MzgxM2M4NG03NzFlb2M5eG86MTU1MTIwODE5NTQyNzpNRVVDSUhSdWFrdmVGZ0RhTTd6UERoMWVEVndZK2NGbXlSYUxWQ054SVRLZGdxTWZBaUVBeGQvOU95NTFOeWxiTWVsRE1ZK0d4S2FzL2Z1TUU2Y0J1bW5JbFBCOXVEVT0%3D&
user_id=k1x4aqdjy3813c84m771eoc9xo&
user_name=tester
```

File: site/content/integrate/slash-commands/_index.md [hunk 2]
```diff
@@ -24,24 +24,26 @@ If you configured a slash command with a trigger word of `test`, a request URL o
 to your Town Square channel, your endpoint will receive:
 
 ```http
-POST /slash-command HTTP/1.1
```
Comment (jasonblais) [inline site/content/integrate/slash-commands/_index.md]: Should we keep the first four lines as-is, or okay to change based on the example you shared?

Comment (crspeller) [reply site/content/integrate/slash-commands/_index.md]: Looks fine with everything changed.

**Pull request merged**